### PR TITLE
block/033 fix

### PR DESCRIPTION
**Pull request by yizhanglinux**

### CODE REVIEW

File: tests/block/033 [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # can be covered
 
 . tests/block/rc
-. common/ublk
+. tests/ublk/rc
```
Comment (kawasaki) [inline tests/block/033:10]: This creates dependency from tests/block to tests/ublk, which is not good. Instead, I suggest to take two actions:
1) move the UBLK_PROG declarations below from tests/ublk/rc to common/ublk
```
if which rublk > /dev/null 2>&1; then
       export UBLK_PROG="src/rublk_wrapper.sh"
else
       export UBLK_PROG="src/miniublk"
fi
```
2) Replace ublk_prog in tests/block/033 with UBLK_PROG.

**Comment by kawasaki** [comment]
Hi @yizhanglinux , thanks for this PR.
As I commented, I suggest some modifications to the first commit. It might be the better to separate it into another PR, and ping @ming1 (Ming Lei) to review.
The other two patches look good to me. The commit for nvme group conflict with my "repeat by conditions" series, but it should be easy to resolve.

File: tests/block/033 (hunk 1 above)
Comment (yizhanglinux) [reply tests/block/033]: Thanks updated as suggested, will open another PR for the misc fix.

Comment (yizhanglinux) [reply tests/block/033]: [@ming1](https://github.com/ming1)  Please help review this change, thanks.

**Comment by kawasaki** [comment]
@yizhanglinux Thanks! This PR has got merged. FYI, I made a follow-up commit for additional fixes.

**Pull request merged**